PodTemplates.d/release-linux.yaml

```diff
@@ -25,30 +25,16 @@ spec:
         privileged: false
         runAsUser: 1000
         runAsGroup: 1000
-  affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-          - matchExpressions:
-              - key: kubernetes.io/os
-                operator: In
-                values:
-                  - linux
-      preferredDuringSchedulingIgnoredDuringExecution:
-        - weight: 1
-          preference:
-            matchExpressions:
-              - key: agentpool
-                operator: In
-                values:
-                  - highmemlinux
+  restartPolicy: "Never"
```

Comment:
what is the rationale behind this change?

Reply:
This isn't a current change, I've just moved this line above the rest.

By the previous comment I was wondering if we should add this restart policy to the packaging pod templates too.
I don't know enough about the release process to know if it's appropriate or not, hence this suggestion.

Reply:
Oh thanks, I missed the line (it's the last one: https://github.com/jenkins-infra/release/pull/348/files#diff-ca89a4622c8149f9ceee5bf387a660eea14ffcb0148c714880be4abc4aff7cb6L54). Now I understand why it's there (I thought it was added).


The restart policy for the release is because it is a non-idempotent process (you can't restart it), while the packaging process is (so we can accept a pod restart if there is node draining).